wire-ios-sync-engine/Source/UnauthenticatedSession/ZMAuthenticationStatus.m

```diff
@@ -316,6 +316,9 @@ - (void)didFailLoginWithEmailBecauseVerificationCodeIsRequired
 - (void)didFailLoginWithEmailBecauseVerificationCodeIsInvalid
 {
     ZMLogDebug(@"%@", NSStringFromSelector(_cmd));
+    if (self.isWaitingForLogin) {
```

Comment:
is it the only (proper) way to stop the loop?

Reply:
What is the loop exactly?

Reply:
added